stable/openvpn/templates/config-openvpn.yaml

```diff
@@ -53,6 +53,35 @@ data:
 
   configure.sh: |-
       #!/bin/sh
+
+      cidr2mask() {
+         # Number of args to shift, 255..255, first non-255 byte, zeroes
+         set -- $(( 5 - ($1 / 8) )) 255 255 255 255 $(( (255 << (8 - ($1 % 8))) & 255 )) 0 0 0
+         [ $1 -gt 1 ] && shift "$1" || shift
+         echo ${1-0}.${2-0}.${3-0}.${4-0}
+      }
+
+      cidr2net() {
+          local i ip mask netOctets octets
+          ip="${1%/*}"
+          mask="${1#*/}"
+          octets=$(echo "$ip" | tr '.' '\n')
+
+          for octet in $octets; do
+              i=$((i+1))
+              if [ $i -le $(( mask / 8)) ]; then
+                  netOctets="$netOctets.$octet"
+              elif [ $i -eq  $(( mask / 8 +1 )) ]; then
+                  echo $octet
```

Comment:
when I run `cidr2net()` locally, my output is:
```
63
10.0.3.0
```
which makes sense, given that my IP is 10.0.3.63. However, I suspect this `echo $octet` might be leftover debug?

Reply:
130fbbb Leftover debug.